omx/gstomxaacdec.c

```diff
@@ -22,12 +22,14 @@
 #endif
 
 #include <gst/gst.h>
-
 #include "gstomxaacdec.h"
 
 GST_DEBUG_CATEGORY_STATIC (gst_omx_aac_dec_debug_category);
 #define GST_CAT_DEFAULT gst_omx_aac_dec_debug_category
 
+#define OMX_AACDEC_INPUT_PORT 0
+#define OMX_AACDEC_OUTPUT_PORT 1
```

Comment:
We could move these two definitions to the helper defs for TI DM81xx section in gstomx.h, and modify the name to GST_OMX_AAC_DEC_ to follow the pattern.

Reply:
66cee90a7d9904dfc8155591da324492d4f520ef